Makefile

```diff
@@ -104,6 +116,7 @@ doc: install_rs_check_toolchain
 bench: install_rs_check_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_CHECK_TOOLCHAIN) bench --bench fft \
 		--features=$(AVX512_FEATURE)
+		--features=$(FFT128_FEATURE)
```

Comment:
this won't work in a Makefile, you are missing a \ to indicate a line break on the previous line, but in that case I wonder if `--features=$(AVX512_FEATURE),$(FFT128_FEATURE)` could work